google/cloud/forseti/services/cli.py

```diff
@@ -233,6 +233,11 @@ def define_server_parser(parent):
               'the server vm or a gcs path starts with gs://).')
     )
 
+    action_subparser.add_parser(
+        'run',
+        help='Run the Forseti process, end-to-end.'
```

Comment:
Please also change the title of this PR to be more descriptive of the work done here, so that when folks are scanning through the PRs, they can easily find the one that they are looking for.  So, something like:
```
Add an endpoint to run the server processes end-to-end.
```

Reply:
I like it.

Reply:
Done